docs/pages/buttons.vue

```diff
@@ -102,58 +92,84 @@
           Avoid ambiguity; be specific about the action that will be performed. For example,
           use 'Save' instead of 'OK'
         </li>
-        <li>Never use commas or other punctuation</li>
-        <li>For raised and flat buttons, always use all-caps text</li>
+        <li>Avoid commas and other punctuation</li>
       </ul>
     </DocsPageSection>
 
     <DocsPageSection title="Placement" anchor="#placement">
       <p>
-        We have some conventions regarding placement of 'raised button' and
-        'flat button' actions:
+        We have some conventions regarding placement of 'raised' and 'flat' buttons:
       </p>
       <ul>
-        <li>Right-aligned in modal pop-ups</li>
-        <li>Left-aligned in page forms</li>
-        <li>Global actions related to tables should be put above the table and right-aligned</li>
+        <li>Right-aligned at the bottom of modal pop-ups</li>
+        <li>Left-aligned a the bottom of page forms</li>
+        <li>Global actions related to full pages should be right-aligned and to the right of the page title on large screens, and left-aligned below the title on small screens. Use of <code>KGrid</code> is recommended</li>
       </ul>
+      <p>
+        Use the <DocsLibraryLink page="KButtonGroup" /> component to add specing between adjacent buttons.
```

Comment:
```suggestion
        Use the <DocsLibraryLink page="KButtonGroup" /> component to add spacing between adjacent buttons.
```

Reply:
lol thanks good catch. I have spec'ing on the brain